### ISSUE TITLE
[MacOS] MikTex Console Crash on Launch: File is empty

### ISSUE THREAD

**Issue by Jacobfaib** [MiKTeX/miktex#535]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md)

TL;DR: macOS Catalina v10.15.4 miktex wont launch even on "fresh" install missing `/Users/myUserName/Library/Application Support/MiKTeX/texmfs/data/miktex/data/le/0997c7e4ae995eb846b56013a9ed6048.fndb-5` **BUT** the file does indeed exist.

`initexmf --report`:
```File is empty.
Data: path="/Users/jacobfaibussowitsch/Library/Application Support/MiKTeX/texmfs/data/miktex/data/le/0997c7e4ae995eb846b56013a9ed6048.fndb-5"
Source: Libraries/MiKTeX/Core/MemoryMappedFile/unx/unxMemoryMappedFile.cpp:107
File is empty.
Info: path="/Users/jacobfaibussowitsch/Library/Application Support/MiKTeX/texmfs/data/miktex/data/le/0997c7e4ae995eb846b56013a9ed6048.fndb-5"
Source: Libraries/MiKTeX/Core/MemoryMappedFile/unx/unxMemoryMappedFile.cpp
Line: 107

Sorry, but "MiKTeX Configuration Utility" did not succeed.
log4cxx: No appender could be found for logger (initexmf).
log4cxx: Please initialize the log4cxx system properly.
```

To reproduce:
- Download MikTex dmg
- Open Miktex
- Error

I am using TexMaker with Miktex to handle the packages installed. I was using Texmaker normally on a project when out of the blue it stops compiling. It complained about missing fonts, and not being able to locate pdflatex. I fixed this by manually setting the following paths to `latex` and `pdflatex` in texmaker config:

`Latex: /Library/TeX/texbin/latex -interaction=nonstopmode %.tex`
`PdfLatex: /Library/TeX/texbin/pdflatex -synctex=1 -interaction=nonstopmode %.tex`

On a hunch I then tried opening miktex to see if it was a faulty package install, however this is when miktex stopped opening for good. I get the following error everytime I try opening the application:

> File is empty.
>`/Users/myUserName/Library/Application Support/MiKTeX/texmfs/data/miktex/data/le/0997c7e4ae995eb846b56013a9ed6048.fndb-5`

Checking that folder produces the following results:

`pdftex`
`mf.base`
`0555d750455acc4dcd50d2f673f6e72d.fndb-5`
`0997c7e4ae995eb846b56013a9ed6048.fndb-5` <----- File exists!
`98fe74e1b5ee98f875065b1594092555.fndb-5`
`976509bce5f7c8fd3176c823500b4fc7.fndb-5`
`0555d750455acc4dcd50d2f673f6e72d.fndb-5.log`
`0997c7e4ae995eb846b56013a9ed6048.fndb-5.log`
`98fe74e1b5ee98f875065b1594092555.fndb-5.log`
`976509bce5f7c8fd3176c823500b4fc7.fndb-5.log`

Not knowing anything about the inner workings of miktex the only theory I could come up with is that the path to the file has a space in it (Application Support), but surely the problem can't be that simple...

**Comment by edocevoli** [comment]
Duplicate of #491
